industry_benchmarks/utils/results_cleanup.py

```diff
@@ -382,8 +382,6 @@ def clean_results(json_files: list[str]) -> None:
             del results["protocol_result"]["data"][result_key][0]["outputs"][
                 "structural_analysis"
             ]
-            del results["protocol_result"]["data"][result_key][0]["inputs"]["stateA"]
```

Comment:
We check on line ~273 if we already cleaned up the file, so if someone ran this script already, either the old or new version, we will skip over the files we have already cleaned since we check for `"structural_analysis" in results["protocol_result"]["data"][result_key][0]["outputs"]`